### DISCUSSION TITLE
Add sunrise, sunset, dusk and sawn

### DISCUSSION THREAD

**Discussion by markandersonus** [Pirate-Weather/pirateweather#144] [Ideas]
Would it be possible to add these: sunrise, sunset, dusk and sawn

**Comment by cloneofghosts** [comment]
You're looking for sunrise, sunset, dusk and dawn times? Sunrise and sunset are existing fields on the daily block https://pirateweather.net/en/latest/API/#sunrisetime and https://pirateweather.net/en/latest/API/#sunsettime

For dawn and dusk you'll want to create an enhancement issue and @alexander0042 can look into adding it for you.

**Comment by cloneofghosts** [comment]
Created this issue for you https://github.com/Pirate-Weather/pirateweather/issues/154
